public/stylesheets/detailed-results.css

```diff
@@ -104,27 +104,81 @@ body {
 
 #game-info-container {
   width: 100%;
+  min-height: 250px;
+  background-color: #333;
 }
 .detailed-results-text {
   flex-direction: column;
-  background-color: #333;
+  overflow-wrap: break-word;
+}
+.game-details-item {
```

Comment:
Let's try to write descriptive class names, rather than `item`, how about `text-wrapper`?